src/content/components/BugList/BugList.js

```diff
@@ -261,4 +519,5 @@ BugList.defaultProps = {
   crossOutResolved: true,
   showResolved: true,
   visibleIfEmpty: true,
+  fetchBugs: () => {},
```

Comment:
I added a prop `fetchBugs` to `BugList` which is a function that is called when a bug's priority or iteration is updated which should fetch the bugs again to receive the new iteration or priority of the updated bug. 

In `OtherView.tsx`, `IterationView.tsx`, and `UnassignedView.tsx`, the `useBugFetcher` hook is used to retrieve bugs. [`useBugFetcher`](https://github.com/mozilla/bugzy/blob/master/src/content/hooks/useBugFetcher.ts#L86) takes a parameter for items to add to its `useEffect` dependency array (`updateOn`), which I could make use of. But I'm not too sure what the best is to translate a button click event to a variable to be placed in a `useEffect` dependency array.

What are your thoughts on the best way to re-trigger these `useBugFetcher`s from within `BugList`? The other files don't use `useBugFetcher` so I can just pass [a function that fetches bugs](https://github.com/mozilla/bugzy/blob/set-bug-iteration-and-priority/src/content/components/FeatureView/FeatureView.js#L595-L597) into `BugList`, but I can't do that here.